roles/StackStorm.st2/defaults/main.yml

```
@@ -34,3 +34,5 @@ st2_auth_username: testu
 st2_auth_password: testp
 # Save credentials in ~/.st2/config file
 st2_save_credentials: yes
+# ST2 packs to be installed (list)
+st2_packs: [azure, jira]
```

```suggestion
st2_packs:
  -  st2
```

Per discussion in previous PR, we want to install only `st2` pack by default: https://github.com/StackStorm/ansible-st2/pull/230#issuecomment-491920683